Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -121,7 +123,11 @@ def _run_repl(
         except subprocess.TimeoutExpired:
             process.kill()
             exit_code = process.wait()
-        return "".join(output), exit_code
+        output = "".join(output)
+        if skip:
+            if "can't use pyrepl" in output:
+                self.skipTest("pyrepl not available")
```

Comment:
Combine into one condition maybe?
```suggestion
        if skip and "can't use pyrepl" in output:
            self.skipTest("pyrepl not available")
```